### PR TITLE
Update changelog and version number

### PR DESCRIPTION
**Pull request by aelanman**
After this, we can tag a release, archive on Zenodo, and finally finish with JOSS.

### CODE REVIEW

**Comment by aelanman** [comment]
@jpober @bhazelton This should be ready now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Changelog
 
-## [Unreleased]
```
Comment (bhazelton) [inline CHANGELOG.md]: We usually leave an unreleased section to allow for new changes to be added.

**Comment by aelanman** [comment]
@bhazelton There was apparently a change in pyuvdata that broke some of the warning catches in the tests. That's fixed now.

**Comment by bhazelton** [comment]
ok, I was working on fixing those and some other issues I found. I'll rebase and just push the other fixes.

**Pull request merged**